miio/device.py

```diff
@@ -71,30 +71,22 @@ def accesspoint(self):
     @property
     def model(self) -> Optional[str]:
         """Model string if available."""
-        if self.data["model"] is not None:
-            return self.data["model"]
-        return None
+        return self.data.get("model")
```

Comment:
Please rebase on top of the current master, #1083 should fix the device info problems.

Reply:
Yes, that commit fixed the problem